fastify.js

```diff
@@ -358,6 +362,15 @@ function build (options) {
     return this._contentTypeParser.hasParser(contentType)
   }
 
+  function handleClientError (e, socket) {
+    const body = JSON.stringify({
+      error: http.STATUS_CODES['400'],
+      message: 'Client Error',
+      statusCode: 400
+    })
+    socket.end(`HTTP/1.1 400 Bad Request\r\nContent-Length: ${body.length}\r\n\r\n${body}`)
```

Comment:
Should perhaps set `content-type: application/json` header here as well as the response body is JSON.